server/config/options.go

```diff
@@ -82,6 +83,11 @@ type MetricsConfig struct {
 	Search SearchMetricsConfig
 }
 
+type DatadogTraceConfig struct {
+	// Global switch
+	Enabled bool `mapstructure:"enabled" yaml:"enabled" json:"enabled"`
```

Comment:
Do we expect other variables here? If not, consider adding it as an item under Tracing.

Reply:
+1 better to just use Tracing if we already have otherwise it may be confusing.

Reply:
For now I will just use tracing enabled, when we will have multiple tracers, we will think about how to configure them.